appveyor.yml

```diff
@@ -40,6 +40,9 @@ platform:
     - x64
 
 install:
+    # Skip if we are not in a PR.
+    - cmd: if not defined APPVEYOR_PULL_REQUEST_NUMBER exit 0
```

Comment:
@msarahan, do you have any idea what I need to do to force AppVeyor to exit here? I thought this would be enough, but it appears not to be the case.